.github/workflows/gradle.yml

```
@@ -33,5 +33,13 @@ jobs:
   GUI-Build:
     runs-on: ubuntu-latest
     steps:
-      - name: Launch GUI
-        run: ./gradlew run --args="--gui assets/dna/random/dnalong.fa"
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)